library/src/dotty/runtime/function/JFunction14.java

```diff
@@ -9,16 +9,4 @@
 public interface JFunction14<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, R> extends scala.Function14<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, R>, java.io.Serializable {
```

Comment:
It doesn't need to be done in this PR, but we could also remove all the interfaces that just contain an empty `$init$` method, I think only https://github.com/lampepfl/dotty/blob/44e6d0f427cf251e67131fed02e1dac46babb537/compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala#L1179-L1181 would need to be adapted to check `arity <= 2`